docs/en/siem/rules-api-overview.asciidoc

```diff
@@ -0,0 +1,81 @@
+[[rule-api-overview]]
+[role="xpack"]
+== Detection engine API
+
+You can create rules that automatically turn events and alerts sent to the
+{siem-app} into signals. These signals are displayed on the Engine Detection 
+page. 
+
+For more information on signals, and the difference between signals, events, 
+and alerts, see <<det-engine-terminology, detection engine terminology>>.
+
+The RESTful API has these endpoints:
```

Comment:
I would remove "RESTful". More of a marketing buzzy term and the API is already explained API endpoint descriptions below.